src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -21,13 +18,11 @@ export type BreadcrumbComponentCSSClasses = ComponentCSSClasses<
 export type BreadcrumbComponentTemplates = Required<BreadcrumbTemplates>;
 
 export type BreadcrumbProps = {
-  items: BreadcrumbItem[];
+  items: BreadcrumbConnectorParamsItem[];
```

Comment:
Open question: wouldn't it be better to have a specific type here. Using the connector type leaks the internal detail this `BreadcrumpProps.items` is the same as `BreadcrumbConnectorParamsItem`, when the widget user doesn't really have to know about it.

Reply:
This is the component, so it's not public anyway :) But I import flow this can better be solved globally